### PR TITLE
fix: toggle button value change bindings

### LINKED ISSUE #520: ToggleButton binding doesn't update
**Describe the bug**
If you remove the console log, then the following conditional does not update.

https://github.com/atlas-health/atlas-navigator-ui/blob/d78f94444f8b0865d2a2801897b99981864505fe/src/ensemble/common/screens/workflow/widgets/workflows/2_drugFlow/2_2_order.yaml#L30-L33

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by sagardspeed2**
## Describe your changes
Fix toggle button value change bindings

## Issue ticket number and link

Closes #520 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] I have added tests
- [x] I have added a changeset `pnpm changeset add`
- [ ] I have added example usage in the kitchen sink app

### CODE REVIEW

File: packages/runtime/src/widgets/ToggleButton.tsx [hunk 1]
```diff
@@ -45,47 +45,72 @@ export type ToggleButtonProps = {
...
-
-        action.callback({
-          [id!]: {
-            value: newValue,
-            setValue,
-          },
-        });
-      }
+      action.callback({
+        [id]: {
```
Comment (evshi) [inline packages/runtime/src/widgets/ToggleButton.tsx:72]: I think we just pass the value here to be consistent with other change widgets.

File: packages/runtime/src/widgets/ToggleButton.tsx [hunk 2]
```diff
@@ -45,47 +45,72 @@ export type ToggleButtonProps = {
...
+  );
+
+  // onchange action handler
   const action = useEnsembleAction(onChange);
   const onChangeCallback = useCallback(
-    (_event: React.MouseEvent<HTMLElement>, newValue: string) => {
+    (newValue: string) => {
       if (!action) {
         return;
       }
```
Comment (evshi) [inline packages/runtime/src/widgets/ToggleButton.tsx:69]: So the issue was we don't update the value if there is no action?

**Review by evshi — APPROVED** [review]

File: packages/runtime/src/widgets/ToggleButton.tsx (hunk 2 above)
Comment (sagardspeed2) [reply packages/runtime/src/widgets/ToggleButton.tsx:69]: Yes

File: packages/runtime/src/widgets/ToggleButton.tsx (hunk 1 above)
Comment (sagardspeed2) [reply packages/runtime/src/widgets/ToggleButton.tsx:72]: @evshi  Should I remove other consistent values except the value ?

Comment (evshi) [reply packages/runtime/src/widgets/ToggleButton.tsx:72]: Yes

**Pull request merged**